framework/source/class/qx/bom/element/Attribute.js

```diff
@@ -333,6 +333,42 @@ qx.Bootstrap.define("qx.bom.element.Attribute",
     },
 
 
+    /**
+     * Serializes an HTML attribute into a writert
```

Comment:
serializes into a *what*? I think that's supposed to say "writer" but I don't understand even that...

Reply:
typo, i'll fix the comment - the `writer` it refers to is the parameter that you call to pass strings to write the attribute definition